espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/drivers/DriverContext.kt

```diff
@@ -0,0 +1,26 @@
+package io.appium.espressoserver.lib.drivers
+
+class DriverContext {
+
+
+    private val composeDriver = ComposeDriver()
+    private val espressoDriver = EspressoDriver()
```

Comment:
these both are espresso driver. I would rather rename it to something like `viewsBasedDriver` and `composeBasedDriver`

Reply:
>these both are espresso driver.

I think `Espresso` and `composeTestRule` are different testing frameworks. Both are instrumentation-based. Overall, this repository will have three underlying testing frameworks 
```
Espresso -> For classic views of the app under test
Uiautomator -> For classic views outside app under test
ComposeTestRule -> For compose views inside the app under test
```
Uiautomator currently is supported in a different way, but can be changed to same way as in this PR

Reply:
Makes sense. I thought first that composeTestRule is a part of Espresso. Now I see these are different. Nevertheless the Appium driver itself is called "espresso", which would be confusing

Reply:
I agree, the driver's name as "espresso" would be confusing. Not sure if there is the possibility to support a new alias, maybe something like 'android_instrumentation' or just 'android'(which will have all possible frameworks :) with the ability to choose)